src/components/atomic/Checkbox.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { InputHTMLAttributes } from "react";
+import classNames from "classnames";
+
+type CheckboxProps = InputHTMLAttributes<HTMLInputElement> & {
+  labelText?: string;
+};
+
+const Checkbox = ({ className = "", ...props }: CheckboxProps) => {
+  const wrapperStyles = classNames({
```

Comment:
No need for classNames if it's just 1 static string--you can put the string directly where the element is